packages/eas-cli/src/project/android/applicationId.ts

```diff
@@ -159,10 +159,10 @@ export function warnIfAndroidPackageDefinedInAppConfigForBareWorkflowProject(
 ): void {
   if (AndroidConfig.Package.getPackage(exp) && !warnPrinted) {
     Log.warn(
-      `Specifying "android.package" in ${getProjectConfigDescription(
+      `Specified value for "android.package" in ${getProjectConfigDescription(
         projectDir
-      )} is deprecated for bare workflow projects.\n` +
-        'EAS Build depends only on the value in the native code. Please remove the deprecated configuration.'
+      )} is ignored if native project folders are detected.\n` +
```

Comment:
maybe 

```suggestion
      )} is ignored because native project folders were detected.\n` +
```

Also, this warning will be printed if the native directory exists and it is not gitignored, I think calling that bare workflow is more precise here, but I do not have a strong opinion on that

```suggestion
      )} is ignored because this is a bare workflow project.\n` +
```

This message has added benefit of pointing out to users that they have a bare project and a lot of them lately are ejecting unknowingly

Reply:
I specifically moved away from the old 'bare workflow' wording and opted for 'native project folders' because running prebuild is not the same as bare workflow. Bare workflow is where you have made manual changes to native files.

Reply:
I agree that running prebuild is not the same as bare workflow, but not in the way you described. Bare workflow is when you run prebuild and android/ios are not gitignored. Also, this issue is present only in bare workflow, not every time you run prebuild.

Reply:
If I init a project, define my identifiers, run `expo prebuild`, then `eas build`, I get this warning. One of the most common causes for confusion is that we advertise `expo prebuild` and `expo run` as something that can repeatedly be run _in the managed worfklow_, so we shouldn't mention 'bare workflow' here at all imho.
Bottom line here is: there's a native folder, so we're using it (unless you manually gitignore or easignore it, which is not by default in our templates). Mentioning 'bare workflow' without saying 'there's a native folder' is confusing for users, our daily support questions make that very clear.